### DISCUSSION TITLE
86Box crashes when trying to insert/swap media (Linux, Qt6)

### DISCUSSION THREAD

**Discussion by PolikarpovMosca** [86Box/86Box#4889] [Q&A] [answered]
I am having an issue with 86Box on Gentoo Linux:

Every time I try to insert any floppy, iso image, etc, 86Box crashes (so making it virtually impossible to install or use most software on my VM).
The crash log reads:

Qt: version 6.7.2, platform "xcb"
qt.gui.imageio: libpng warning: IDAT: extra compressed data
*** buffer overflow detected ***: terminated
zsh: IOT instruction  86Box -p 486

I got a second machine with Arch Linux, which works fine. The only difference is that Arch has a slightly newer Qt version 6.7.3
tried compiling 86Box on Gentoo without Qt support (hoping there was some fallback toolkit or framework instead of Qt, but unsurprisingly I just got a working 86Box, with no menu support and thus no way of inserting or removing media).

It did work with Qt5, and I think that older versions of Qt6

**Comment by OBattler** [comment, marked as answer]
Use the latest 86Box build.

**Reply by PolikarpovMosca** [reply]
Sorry, forgot to specify that: Both are 4.2.1, which I think is the latest version

**Reply by TC1995** [reply]
By latest, he meant the latest nightly ones based.

**Reply by PolikarpovMosca** [reply]
Thanks. I can confirm that the bug doesn't happen with the nightly version. I can use that while I wait for a stable update

**Comment by PolikarpovMosca** [comment]
Ah, adding a note: Same type of crash while trying to save changes on the sound department. I had standalone MPU-401 somehow enabled, which is not seemingly valid for a sb16 (ISA16). and MT32 emul. After manually editing the config file to remove MT32 and this MPU option, now it seems to also work...
